tests/rbd/readonly_namespace_image_migration.py

```diff
@@ -47,7 +48,7 @@ def image_migrate_and_verify(
     src_spec1 = src_pool + "/" + namespace + "/" + image
     dest_spec1 = dest_pool + "/" + image
     client_id = "client.1"
-    mgr_role = "profile rbd-read-only"
+    mgr_role = f"profile rbd-read-only pool={src_pool} namespace={namespace}"
```

Comment:
IMO, we could keep namespace separate
(Got the answer for one of my question above)

Reply:
we are forming the role as needed, so i dont think so we need to keep the namespace/pool seperately.

Reply:
Resolved as per automation peer meeting discussion.